firstpost_processor.py

```diff
@@ -26,12 +26,16 @@ def download(self):
         cid_count = len(cids)
         idx = 1
         for cid in cids:
-            print("Getting messages for {}/{} {} - {}".format(idx,
-                                                              cid_count, cid, self.channel.get(cid)))
-            gotit = self.fp.get_channel(cid)
-            if gotit:
-                print("\tSkipping channel -- already got it")
+            channel_entry = self.channel.get(cid)
+            if channel_entry:
+                name = channel_entry['friendly_name']
+            else:
+                name = "Unknown"
+            if name != "announcements":
```

Comment:
Will you please help me understand why we'd move on if the channel is #announcements?

Reply:
That was for testing, and should be out of the final version of the code.

Reply:
Oh, no, actually sorry it was still there -- submitted new change to take it out